sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/data/SpanData.java

```diff
@@ -340,12 +340,19 @@ static Status create(StatusCode code, @Nullable String description) {
       return ImmutableStatus.create(code, description);
     }
 
+    /** Returns the status code. */
+    StatusCode getStatusCode();
+
     /**
-     * Returns the canonical status code.
+     * Returns the status code.
      *
-     * @return the canonical status code.
+     * @deprecated Please use {@link #getStatusCode()}. This method will be removed in the next
+     *     release.
      */
-    StatusCode getCanonicalCode();
+    @Deprecated
```

Comment:
I think we currently still have enough other breaking changes that it is OK to completely remove this. It's just a renaming, right?

Reply:
I've started thinking that since we're moving to a 2-week release cycle, that it would be more friendly to end users to introduce deprecations that are then deleted in the following release. We discussed this at the SIG meeting this morning, and it was generally agreed to.  This seemed like a easy way to ease into the practice. :)